src/modelfree/configs/ray/aws.yaml

```diff
@@ -2,19 +2,20 @@ cluster_name: aprl
 min_workers: 0
 max_workers: 10
 initial_workers: 0
-target_utilization_fraction: 0.8
+target_utilization_fraction: 0.4
```

Comment:
Why do we want our cluster to be 60% idle?

Reply:
This was from when I thought the problem with number of CPUs not going up was that it somehow was just under the utilization fraction, and I was putting this down to see if there'd be new workers created. Meant to revert it, just forgot to.